spec/ndx-microscopy.namespace.yaml

```
@@ -8,6 +8,8 @@ namespaces:
   - cody.baker@catalystneuro.com
   - alessandra.trapani@catalystneuro.com
   schema:
-  - namespace: core
+  - namespace:
+    - core
+    - ndx-ophys-devices
```

Should be:
```suggestion
  - namespace: core
  - namespace: ndx-ophys-devices
```

`schema` should be a list of dicts with certain keys

---

In general, I recommend using the API to create the specs, though I know it adds another step.